doc/swagger.v1.yml

```diff
@@ -285,6 +310,51 @@ definitions:
         format: date-time
         description: when the transaction was entered
     required: ['id', 'user_id', 'user_name', 'amount', 'note', 'created_at']
+    
+  FinancialSummary:
+    type: object
+    properties:
+      id:
+        type: integer
+      user_id:
+        type: 'integer'
+        default: null
+        description: id of user who entered the transaction (may be <tt>null</tt> for deleted users or 0 for a system user)
+      user_name:
+        type: 'string'
+        default: null
+        description: name of user who entered the transaction (may be <tt>null</tt> or empty string for deleted users or system users)
+      oder_id:
+        type: 'integer'
+        default: null
+        description: id of order group, user is assigned to
+      credit:
+        type: number
+        default: 0
+        description: actual available credit of user/ordergroup
+      balance:
+        type: 'integer'
+        default: 0
+        description: credit + open claim
+      classes:
+        type: array
+        default: null
+        description: array of transaction classes and their credit, null if no transaction classes are defined
+        items:
+          $ref: '#/definitions/FinancialClass'
+          
+  FinancialClass:
+    type: object
+    properties:
+      id:
+        type: integer
+        description: id of transaction class
+      name:
+        type: string
+        description: name of transaction class
+      credit:
+        type: number
+        description: actual credit of transaction class
```

Comment:
Ok, I'm not intimately familiar with how financial transaction classes function, but part of me expects the same fields as above: so `account_balance` as well as `available_funds`. But that doesn't really make sense, since you only have `available_funds` for one financial transaction class, from my limited understanding.

Sorry for the confusion, we somehow need to get this right. Do you have more background in financial transaction classes to explain it a bit?